index.html

```diff
@@ -191,35 +191,35 @@ <h1>Stay tuned for Technica 2020!</h1>
       <div class="heading">
         <h2> Explore Our Past Events </h2>
       </div>
-      <a class="past-link mt-3" href="https://2019.gotechnica.org/">
+      <a class="past-link mt-3" target="_blank" href="https://2019.gotechnica.org/">
         <img src="./img/2019_img.jpg">
         <div class="overlay">
           <h3>2019</h3>
           <h2>Go Beyond</h2>
         </div>
       </a>
-      <a class="past-link" href="https://2018.gotechnica.org/">
+      <a class="past-link" target="_blank" href="https://2018.gotechnica.org/">
         <img src="./img/2018_img.jpg">
         <div class="overlay">
           <h3>2018</h3>
           <h2>Light The Way</h2>
         </div>
       </a>
-      <a class="past-link" href="https://2017.gotechnica.org/">
+      <a class="past-link" target="_blank" href="https://2017.gotechnica.org/">
         <img src="./img/2017_img.jpg">
         <div class="overlay">
           <h3>2017</h3>
           <h2>Three Year Anniversary</h2>
         </div>
       </a>
-      <a class="past-link" href="https://web.archive.org/web/20161114145145id_/http://gotechnica.org/">
+      <a class="past-link" target="_blank" href="https://2016.gotechnica.org/">
```

Comment:
not sure why, but https://2016.gotechnica.org is not connecting for me :( 
could you change this to http://2016.gotechnica.org instead?